cupyx/scipy/sparse/linalg/_iterative.py

```diff
@@ -7,23 +7,29 @@
 from cupy_backends.cuda.libs import cusparse as _cusparse
 from cupy_backends.cuda.libs import cublas as _cublas
 from cupyx.scipy.sparse import csr
+from ._interface import aslinearoperator, LinearOperator, IdentityOperator
```

Comment:
In our project, relative import and non-module import is prohibited.
```suggestion
from cupyx.scipy.sparse.linalg import _interface
```

Reply:
Thanks! Will fix in that way.